packages/puppeteer-crawler/src/internals/puppeteer-launcher.ts

```diff
@@ -77,15 +77,24 @@ export interface PuppeteerLaunchContext extends BrowserLaunchContext<PuppeteerPl
      */
     launcher?: unknown;
 
+    /**
+    * With this option selected, all pages will be opened in a new incognito browser context.
+    * This means they will not share cookies nor cache and their resources will not be throttled by one another.
+    * @default false
+    */
+    useIncognitoPages?: boolean;
+
     /**
      * This setting hides most of the known properties that identify headless Chrome and makes it nearly undetectable.
      * It is recommended to use it together with the `useChrome` set to `true`.
+     * @deprecated
```

Comment:
just a head up, this will be removed in crawlee altogether (but lets keep it like this for now)